CONTRIBUTING.md

```diff
@@ -22,6 +22,15 @@ A few key points to take into account:
 * Add a blank line before return statements, unless the return is alone inside a statement-group (like an if statement);
 * Use braces to indicate control structure body regardless of the number of statements it contains;
 
+Please use [php-cs-fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer) before submitting your code for review. This tool will reformat all of your code to follow our coding standards.
+
+The rules we follow are formalized in a .php_cs file at the root of the project. So as to use our ruleset, you have to:
+1. be at the root of the project
```

Comment:
You need to leave an empty line before this one to make this a list. Nit: start with uppercase, add period at the end.